CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 #### Enhancements
 
+* Prevent from compiling `SwiftLint` target when only using `SwiftLintPlugin` on macOS.
```

Comment:
Please append two spaces to the end of the description. They are required to enforce a line break in the rendered Markdown.

Reply:
done

Thank you for the tip, now I know.